cli/listener/src/main.rs

```diff
@@ -36,24 +45,13 @@ pub struct Opts {
     #[clap(long, default_value = "300")]
     poll_frequency_seconds: u32,
 
-    /// Location of the SQLite DB file.
+    /// Location of the SQLite DB file, defaults to "listener.db".
     #[clap(long, default_value = "listener.db")]
     db_path: String,
-}
 
-struct State {
-    pub solido: Lido,
-}
-
-impl State {
-    pub fn new(
-        config: &mut SnapshotConfig,
-        solido_program_id: &Pubkey,
-        solido_address: &Pubkey,
-    ) -> Result<Self, SnapshotError> {
-        let solido = config.client.get_solido(solido_address)?;
-        Ok(State { solido })
-    }
+    /// Listen address and port for the http server that serves a /metrics endpoint. Defaults to 0.0.0.0:8923.
+    #[clap(long, default_value = "0.0.0.0:8923")]
```

Comment:
We should pick a different default port than the one used by `solido run-maintainer`. How about 8929, the next prime after 8923? :grimacing:

Reply:
ohhh... that, definitely 😬